compliance/README.md

```diff
@@ -0,0 +1,22 @@
+# Compliance Automation Engineer Challenge
+
+At Tanda, we maintain a number of templates for [Modern Awards](https://www.fairwork.gov.au/awards-and-agreements/awards/list-of-awards) which our customers can apply automate their timesheet calculations. A key part of maintaining these templates is documenting the miniminum wages that can be paid as part of these awards, as well as any age-based adjustments to these.  
```

Comment:
which our customers can apply automate their timesheet calculations.
should probably be
which our customers can apply to automate their timesheet calculations.